app/src/services/api/api.ts

```diff
@@ -0,0 +1,37 @@
+import { api as generatedApi } from "./generated/api.generated";
+
+export const api = generatedApi.enhanceEndpoints({
```

Comment:
is this generated ?

Reply:
nop, it's an example on how to configure endpoints to handle revalidation : https://rtk-query-docs.netlify.app/concepts/mutations#advanced-mutations-with-revalidation